index.js

```diff
@@ -2382,28 +2382,22 @@
     impl: fromRight
   };
 
-  //# fromEither :: b -> Either a b -> b
+  //# fromEither :: Either a a -> a
   //.
-  //. Takes a default value and an Either, and returns the Right value
-  //. if the Either is a Right; the default value otherwise.
-  //.
-  //. The behaviour of `fromEither` is likely to change in a future release.
-  //. Please use [`fromRight`](#fromRight) instead.
+  //. Takes an Either with the same type on the Left and on the Right
+  //. and return whichever value exists.
```

Comment:
We could add the following sentence:

> The inverse of [`tagBy`][1].


[1]: https://sanctuary.js.org/#tagBy

Reply:
Would you mind adding this sentence, @futpib?